lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/lib/factory.js

```diff
@@ -23,7 +23,24 @@
 var constantFunction = require( '@stdlib/utils/constant-function' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var exp = require( '@stdlib/math/base/special/exp' );
-var pow = require( '@stdlib/math/base/special/pow' );
+var expm1 = require( '@stdlib/math/base/special/expm1' );
+
+
+// FUNCTIONS //
+
+/**
+* Helper function for repeated computation in the MGF formula.
+*
+* @private
+* @param {number} x - input value
+* @returns {number} evaluated result
+*/
+function phi2( x ) {
```

Comment:
For the JS implementation, I suggest moving `phi2` to a separate file and then importing in both this file and in `main.js`. This will reduce code duplication.